### PR TITLE
Change storage values size to int64_t

### PR DESCRIPTION
**Pull request by infernumx**
Main point of this was to use `os.mtime()` as a storage value for more precision.

### CODE REVIEW

**Comment by andersonfaaria** [comment]
In order for my antibot work I had to store the os.mtime() in some specific situations, what I did to workaround this was to make some math in order reduce the size of the number and be able to still store miliseconds without having to perform such an aggressive change.

One could simply use modulo operator in order to achieve that, but another simply way is to collect the os.time() once and store it forever as a global storage value, it will take a couple decades to explode if you use sugarcoat a function to consider os.time() - globalStorage with os.time()

Besides this 'workarounds', your PR is looking really neat but I would suggest we change it for string instead and try to tackle #1545

File: data/migrations/26.lua [hunk 1]
```diff
@@ -0,0 +1,3 @@
+function onUpdateDatabase()
+	return false
+end
```
Comment (DSpeichert) [inline data/migrations/26.lua:3]: missing EOL

File: schema.sql [hunk 2]
```diff
@@ -312,7 +312,7 @@ CREATE TABLE IF NOT EXISTS `player_spells` (
 CREATE TABLE IF NOT EXISTS `player_storage` (
   `player_id` int(11) NOT NULL DEFAULT '0',
   `key` int(10) unsigned NOT NULL DEFAULT '0',
-  `value` int(11) NOT NULL DEFAULT '0',
+  `value` bigint(20) NOT NULL DEFAULT '0',
```
Comment (DSpeichert) [inline schema.sql:315]: You make it inconsistent by having a schema snapshot with latest change but without actually upgrading the version number in the schema. You don't really need to edit this file.

**Comment by Zbizu** [comment]
You can store those values in server memory(`antibotLastCheck[cid] = os.mtime()`). This isn't really something we need to save like we do with quest states.

**Comment by DSpeichert** [comment]
It's been 10 days without any response, are you still interested @infernumx ?

**Pull request closed without merging**